projects/ng-devtools-backend/src/lib/directive-forest/ltree.ts

```diff
@@ -71,24 +71,26 @@ export class LTreeStrategy {
     const tNode = data[idx];
     const node = lView[idx][ELEMENT];
     const element = (node.tagName || node.nodeName).toLowerCase();
-    for (let i = tNode.directiveStart; i < tNode.directiveEnd; i++) {
-      const instance = lView[i];
-      const dirMeta = data[i];
-      if (dirMeta && dirMeta.template) {
-        component = {
-          name: element,
-          isElement: isCustomElement(node),
-          instance,
-        };
-      } else if (dirMeta) {
-        directives.push({
-          name: getDirectiveName(instance),
-          instance,
-        });
+    if (tNode) {
```

Comment:
Can we have an early return instead to avoid nesting?

Reply:
I can do. I did consider that, but wanted to avoid repeating the output structure:
```TS
{
  nativeElement: node,
  children: [],
  element,
  directives,
  component,
};
```
... might be better to be explicit about what is returned:
```TS
{
  nativeElement: node,
  children: [],
  element,
  directives: [],
  component: null,
};
```

Reply:
Repeating should be fine given the type system will catch if we break the shape.

Reply:
Done!